src/tools/cephfs/Resetter.cc

```diff
@@ -101,34 +135,24 @@ int Resetter::reset(mds_role_t role)
   if (r != 0) {
     return r;
   }
-
-  r = _write_reset_event(&journaler);
-  if (r != 0) {
-    return r;
+ 
+  if (is_mdlog) {
+    r = _write_reset_event(&journaler); // reset envent is specific for mdlog journal
+    if (r != 0) {
+      return r;
+    }
   }
-
   cout << "done" << std::endl;
 
   return 0;
 }
 
-int Resetter::reset_hard(mds_role_t role)
+int Resetter::reset_hard()
 {
   auto fs =  fsmap->get_filesystem(role.fscid);
-  assert(fs != nullptr);
-  int64_t const pool_id = fs->mds_map.get_metadata_pool();
-
-  JournalPointer jp(role.rank, pool_id);
-  jp.front = role.rank + MDS_INO_LOG_OFFSET;
-  jp.back = 0;
-  int r = jp.save(objecter);
```

Comment:
Removing this `int r` causes a compiler error:

https://jenkins.ceph.com/job/ceph-dev-new-build/ARCH=x86_64,AVAILABLE_ARCH=x86_64,AVAILABLE_DIST=centos7,DIST=centos7,MACHINE_SIZE=huge/9674//consoleFull

````
/home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos7/DIST/centos7/MACHINE_SIZE/huge/release/13.0.1-2030-g720fb5a/rpm/el7/BUILD/ceph-13.0.1-2030-g720fb5a/src/tools/cephfs/Resetter.cc: In member function 'int Resetter::reset_hard()':
/home/jenkins-build/build/workspace/ceph-dev-new-build/ARCH/x86_64/AVAILABLE_ARCH/x86_64/AVAILABLE_DIST/centos7/DIST/centos7/MACHINE_SIZE/huge/release/13.0.1-2030-g720fb5a/rpm/el7/BUILD/ceph-13.0.1-2030-g720fb5a/src/tools/cephfs/Resetter.cc:169:3: error: 'r' was not declared in this scope
   r = cond.wait();
   ^
````

Reply:
sorry, i should compile it locally first

Reply:
fixed